### ISSUE TITLE
html reader color style not working (related to closed #451)

### ISSUE THREAD

**Issue by ShaunWolfe** [PHPOffice/PhpSpreadsheet#838]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
reading html file with style="color:#FF00000" should affect the text color

### What is the current behavior?
text color is not affected. Remains default black

### What are the steps to reproduce?
Simply create a file with html code using style="color:#anycolor"  -- this may be related to just `<th>` or `<td>`, however.

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

require __DIR__ . '/vendor/autoload.php';


// create some offending HTML and save it to a file.
$testHTML =
    "<table><tr>".
    "<td style='color:#FF0000'>This should be Red</td>".
    "<td style='background-color:#FF0000; color:#FFFFFF'>This should white on Red</td>".
    "</tr></table>";
$handle = fopen("htmlfile","w+");
fwrite($handle, $testHTML);
fclose($handle);

// Create new Spreadsheet object using Html reader
    $reader = new \PhpOffice\PhpSpreadsheet\Reader\Html();
    $spreadsheet = $reader->load("htmlfile");
    $writer = PhpOffice\PhpSpreadsheet\IOFactory::createWriter($spreadsheet, 'Xlsx');
    $writer->save('colorbug.xlsx');

// resulting colorbug.xlsx file will have 
//     A1 -> black-on-white;   A2 -> black-on-red.
// it should have 
//     A1 -> red-on-white;  A2 -> white-on-red.
// 
```

### Which versions of PhpSpreadsheet and PHP are affected?
1.6.0

### Additional Information
[This related, I believe, to a closed bug reported in April 2018](https://github.com/PHPOffice/PhpSpreadsheet/issues/451)

**Comment by patrickbrouwers** [comment]
@ShaunWolfe for me the above code works. 

![screenshot 2019-01-28 at 17 52 21](https://user-images.githubusercontent.com/7728097/51852046-8134e400-2325-11e9-81b1-8bcc296b2467.png)

**Comment by ShaunWolfe** [comment]
@patrickbrouwers Thank you for taking a look. I had a colleague try the same code and it did not work for him either.  Therefore, I suspect it's environmental or some configuration.  Can you tell me what version of PhpSpreadsheet you are using?

**Comment by patrickbrouwers** [comment]
@ShaunWolfe I'm on 1.6 yes.

**Comment by ShaunWolfe** [comment]
Verified that this is fixed between 1.4 and 1.6. Thanks @patrickbrouwers !
